docs/docs/README.md

```diff
@@ -4,7 +4,7 @@
 
 !!! warning "This project is still under development and is not officially supported."
 
-[Atlassian DC Apps program](https://developer.atlassian.com/platform/marketplace/dc-apps-submitting-your-app/#step-2--test-your-app-s-performance-at-scale)
+The [Atlassian DC Apps program](https://developer.atlassian.com/platform/marketplace/dc-apps-submitting-your-app/#step-2--test-your-app-s-performance-at-scale){.external}
 provides App vendors in Atlassian ecosystem with tools to setup ready-to-use environment. 
 This project provides a tool to provision infrastructure for Atlassian DC helm chart products.
 At this stage the scope is providing the infrastructure for Bamboo DC.
```

Comment:
Dont know if this really captures the essence of the Terraform project. Im a bit confused with DCAPT being mentioned, does it play a role in provisioning infra with this project?